datacenter/dtr/2.0/high-availability/backups-and-disaster-recovery.md

```diff
@@ -86,6 +86,7 @@ $ tar -tf /tmp/backup.tar
 
 ## Restore DTR data
 
+DTR data must be restored to the same release version where the backups were obtained.
```

Comment:
The sentence is a bit confusing 😢 Can you rephrase to something like:
"When restoring, make sure you use the same version of the `docker/dtr` image that you've used to create the backup".